src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -81,6 +81,8 @@ public enum Feature {
         @JEP(number=478, title="Key Derivation Function API", status="Preview")
         KEY_DERIVATION,
         LANGUAGE_MODEL,
+        @JEP(number = 502, title = "Stable Values", status = "Preview")
+        STABLE_VALUES,
```

Comment:
Please add your enum constant above `LANGUAGE_MODEL` - it is a reserved key for certain reflective preview APIs.